integrationtests/testdata/issueTokenAndSetRoles/token-after-set-roles-and-issue.json

```diff
@@ -1,22 +1,22 @@
 {
   "ticker": "SEMI",
-  "currentOwner": "61646472",
+  "currentOwner": "erd1k04pxr6c0gvlcx4rd5fje0a4uy33axqxwz0fpcrgtfdy3nrqauqqgvxprv",
```

Comment:
Why didn't you keep consistency in using different addresses?

Reply:
It is just an address for testing, I don't need to be consistent in what addresses I use.